frontend/lib/requests/lesson.ts

```
@@ -37,3 +45,273 @@
+  id: number,
+  data: Partial<z.infer<typeof LessonSchema>>,
+  options: { reload?: boolean; regenerateSlug?: boolean } = {
+    reload: false,
+    regenerateSlug: false,
+  },
+) {
+  try {
+    if (data.blocks) {
+      data.blocks = data.blocks.map(({ id, ...rest }) => rest);
+    }
+    const dataToSend: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)